Assets/Scripts/Robot/IRobotController.cs

```diff
@@ -223,5 +223,7 @@ public DetectedWall(float distance, float relativeAngle)
         /// </summary>
         /// <returns>a list of relative positions (containing integer ids of sensed robots) </returns>
         SensedObject<int>[] SenseNearbyRobots();
+
+        Bitmap CalculateCommunicationZone(Vector2Int position);
```

Comment:
You probably don't want this any longer?

Reply:
yep, thats a leftover from a previous iteration. its been removed